### ISSUE TITLE
Deploy not including static.json in shared

### ISSUE THREAD

**Issue by dawnerd** [architect/deploy#43]
**Describe the issue**
When running a deploy with static fingerprinting, the static.json does not end up as part of the bundle and is missing from shared.

**Steps to reproduce**
1. Enable asset fingerprinting
2. Make sure no static.json exists
3. deploy production
4. Logs show static.json uploaded to s3
5. Check lambda function, verify static.json doesn't exist in shared

**Comment by jgallen23** [comment]
some more context: https://twitter.com/greg_allen/status/1179448002248765440

**Comment by ryanblock** [comment]
Just to confirm: this is Arc 6, yes?

**Comment by jgallen23** [comment]
yep

**Comment by jgallen23** [comment]
before deploy:
```
» find . | grep static.json                                                                                                                                                                 13:20:21
./node_modules/@architect/hydrate/src/shared/copy-static-json.js
```

after `arc deploy`
```
» find . | grep static.json                                                                                                                                                                 13:20:26
./node_modules/@architect/hydrate/src/shared/copy-static-json.js                                                                                                                                    
./public/static.json
```

after `arc deploy and then arc hydrate`
```
» find . | grep static.json                                                                                                                                                                 13:21:06
./src/http/get-index/node_modules/@architect/shared/static.json                                                                                                                                     
./src/http/get-sprite_symbol_svg/node_modules/@architect/shared/static.json                                                                                                                         
./node_modules/@architect/hydrate/src/shared/copy-static-json.js                                                                                                                                    
./public/static.json 
```

**Comment by jgallen23** [comment]
seems like static.json needs to be generated _before_ hydrating lambdas

**Comment by ryanblock** [comment]
Yeah, I can see how in our order of operations this got missed. I'll work on a fix, shouldn't take super long. Please stand by, will post an RC to play around with asap!

**Comment by ryanblock** [comment]
Ok, please `npm i @architect/architect@6.0.13-RC.1` and let me know if that fixes things for you! Verified here but always good to get more eyes.

**Comment by dawnerd** [comment]
Looks like that did the trick. Thanks for the fast turn-around!

**Comment by ryanblock** [comment]
No sweat! Just shipped to mainline as `6.0.13`. Thanks for helping me pinpoint this issue so quickly!
